sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -158,7 +158,8 @@ class DataSourceV2SQLSuite
   test("Describe column for v2 catalog") {
```

Comment:
can we test this specific change in `CharVarcharDDLTestBase`?

Reply:
Thanks for the reminder, there's a todo